src/simulations/local_elec.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, prm):
         self.n_homes = prm['syst']['n_homes']
         self.homes = range(self.n_homes)
 
-        if self.prm['grd']['manage_voltage']:
+        if self.prm['grd']['manage_voltage'] or self.prm['grd']['manage_agg_power']:
```

Comment:
no let's not create the whole network object just to manage agg power. This is unnecessary.